src/services/apis/bbga.js

```diff
@@ -67,6 +64,25 @@ export async function getMeta (variableName) {
   return meta[variableName.toUpperCase()]
 }
 
+/**
+ * This is a fixed static url published in dcatd. The maintainers can publish updates of the data in
+ *  https://data.amsterdam.nl/datasets/G5JpqNbhweXZSw/basisbestand-gebieden-amsterdam-bbga/
+ *
+*/
+const STD_DATAL_LOCATION_URL = 'https://api.data.amsterdam.nl/dcatd/datasets/G5JpqNbhweXZSw/purls/3'
+
+/**
+ * Gets all the std values. This is an fixed url published in dcatd.
```

Comment:
The proper abbreviation for standard deviation is either 'Std Dev' or 'SD'. 'STD' is something completely different and I wouldn't use it in the context of this web application.